tools/build_config/build.ps1

```diff
@@ -188,6 +192,43 @@ function Invoke-Package {
   }
 }
 
+function Invoke-Build-Docs {
+  (Get-Content "./build/CPackConfig.cmake" | Where-Object {$_ -match 'CPACK_PACKAGE_FILE_NAME'}) -match '.*"Horace-([^"]+)".*'
+  $build_id = $Matches[1]
+  (Get-Content ./documentation/user_docs/docs/conf.py) | Foreach-Object {$_ -replace 'release = .*', "release = '${build_id}'"} | Set-Content  ./documentation/user_docs/docs/conf.py
+
+  ./documentation/user_docs/make.bat html
+  ./documentation/user_docs/make.bat html
+  # Undo change to allow checkout
+  git checkout ./documentation/user_docs/docs/conf.py
+
+  Foreach ($f in Get-ChildItem -Path './documentation/user_docs/build/html' -Filter *.html) {
+      (Get-Content "./documentation/user_docs/build/html/$f") | Where-Object {$_ -notmatch '\\[NULL\\]'} | Set-Content "./documentation/user_docs/build/html/$f"
+  }
+  Compress-Archive -Path ./documentation/user_docs/build/html/* -DestinationPath "./docs.zip"
+  if ($LASTEXITCODE -ne 0) {
+    exit $LASTEXITCODE
+  }
+}
+
+function Invoke-Push-Docs {
+  git config --local user.name "PACE CI Build Agent"
+  git config --local user.email "pace.builder.stfc@gmail.com"
```

Comment:
should the username and url be passed in rather than hard coded in the script?

Reply:
Possible, I suppose, but it's hardcoded in the API token anyway.